src/transactions/utils.js

```diff
@@ -35,6 +35,19 @@ const prepareTransaction = (transaction, secret, secondSecret) => {
 	return transactionWithId;
 };
 
+const isValidValue = (value) => {
+	if (
+		// eslint-disable-next-line no-self-compare
+		value !== value
```

Comment:
Is this for `NaN`? Prefer explicit checks. Or even
```js
if ([undefined, false, NaN, /* ... */].includes(value)) {
    / ...
}
```

Reply:
Actually, since this is just a bunch of boolean checks it should probably return a value directly rather than having a body at all.